Reflections/R4 - May 7.md

```diff
@@ -0,0 +1,15 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+The biggest thing I accomplished this week is I put in place a structure menuing system.  It uses a newly-created dynamic button object to create control what different structures can make.
+
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+Full disclosure this is has been quantifiably one of the worst weeks of my life, and as such I wasn't able to get as much done on the project as I would have liked. I didn't really have any problems with the program or the class however, I simply had most of my time taken up by a couple of personal issues.
```

Comment:
Incredibly fair on all accounts. The prospect of graduation draws ever closer... oof!